config/nginx.conf.development.template

```diff
@@ -179,4 +179,9 @@ http {
       }
     }
   }
+  server {
```

Comment:
Can you tell me about the idea behind this?  What's our eventual goal for exposing the docs, in both dev and production (i.e., what are the URLs)?  cc @cindyledev

Reply:
I initially just wanted to expose the docs via `/docs` route